src/phoenix/experimental/evals/utils.py

```diff
@@ -0,0 +1,33 @@
+"""
+Utility functions for evaluations.
+"""
+
+import json
+from io import BytesIO
+from urllib.error import HTTPError
+from urllib.request import urlopen
+from zipfile import ZipFile
+
+import pandas as pd
+
+
+def download_benchmark_dataset(task: str, dataset_name: str) -> pd.DataFrame:
```

Comment:
just an idea: these two arguments can be combined as one e.g. "task/dataset_name" and that would not affect the http address

Reply:
Oh shoot, missed this comment, but that's a good idea.